app/controllers/pokemon_controller.rb

```diff
@@ -39,6 +41,14 @@ def update
     end 
   end
 
+  def destroy
+    @pokemon = Pokemon.find(params[:id])
+    pokemon_name = @pokemon.nickname
+    @pokemon.destroy
```

Comment:
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔
SHAME 🔔 🔔

(You should always use the bang version if you're not checking the return value)